### PR TITLE
"As is" user types

### PR DESCRIPTION
**Pull request by molaux**
Here is my proposal to handle types like `JSONB` or `BOOLEAN` (handled by Sequelize/Postgres) without conversion.
Not sure it is relevant, so let me know if there is an other way to go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,17 @@ General options applied to all formatter.
...
     Default is `false`.
+  
+  * `asIsUserDatatypePrefix`
+
+    This option gives the ability to bypass user type convertion and forward user type to models, as is. 
+    
+    For example, if defined to `_`, and a user type 
+    `_BOOLEAN` is defined in Mysql Workbench (resolving to `TINYINT(1)`, for example, but no matter), then the model field will have `BOOLEAN`. 
+    
+    It is usefull for non Mysql types like `BOOLEAN` or `JSONB` that certain ORMs can handle.
```
Comment (tohenk) [inline README.md:161]: Mysql should be MySQL.

File: lib/Formatter/FormatterInterface.php [hunk 2]
```diff
@@ -58,6 +58,7 @@ interface FormatterInterface {
     const EOL_WIN                           = 'win';
     const EOL_UNIX                          = 'unix';
 
+    const CFG_AS_IS_USER_DATATYPE_PREFIX    = 'asIsUserDatatypePrefix';
```
Comment (tohenk) [inline lib/Formatter/FormatterInterface.php:61]: Add a blank line after this.

File: lib/Model/Document.php [hunk 3]
```diff
@@ -218,10 +218,10 @@ protected function loadUserDatatypes()
 
         foreach ($userTypes as $userType) {
             $userTypeName = $userType->xpath("value")[2];
```
Comment (tohenk) [inline lib/Model/Document.php:220]: Cast to string here, so the next code doesn't need to do casting anymore.

**Comment by molaux** [comment]
Thank you for the code review.

**Pull request merged**